content/getting-started/modifying-access.md

```diff
@@ -5,5 +5,5 @@ weight: 8
 aliases:
 ---
 
-When you have moved your repository or need to update the username/password or private key used to access it, you can change the access details in **App settings > Repository settings > Repository access settings**. You can also generate a new private key. Once you have changed the relevant settings, click **Update access settings**. Codemagic will verify it has access to the repository using the updated details before saving the settings.
+When you have moved your repository or need to update the username/password or private key used to access it, you can change the access details in **App settings > Repository settings > Repository access settings**. **Repository access settings** will be visible only if apps are added through custom sources. You can also generate a new private key in the settings section. Once you have changed the relevant settings, click **Update access settings** and then Codemagic will verify that it has access to the repository using the updated details before saving the settings.
```

Comment:
- will be visible > will be present
- "custom sources" is a term we use internally. A more accurate phrasing is "added via URL using HTTP/HTTPS or SSH keys"